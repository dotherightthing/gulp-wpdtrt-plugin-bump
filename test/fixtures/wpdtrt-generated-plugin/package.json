{
  "name": "wpdtrt-generated-plugin",
  "version": "0.12.345"
}
