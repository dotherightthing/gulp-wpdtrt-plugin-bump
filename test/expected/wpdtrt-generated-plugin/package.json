{
  "name": "wpdtrt-generated-plugin",
  "version": "2.0.0"
}